docs/source/en/index.md

```diff
@@ -465,6 +466,7 @@ Flax), PyTorch, and/or TensorFlow.
 |             UMT5              |       ✅        |         ❌         |      ❌      |
 |           UniSpeech           |       ✅        |         ❌         |      ❌      |
 |         UniSpeechSat          |       ✅        |         ❌         |      ❌      |
+|            UnivNet            |       ❌        |         ❌         |      ❌      |
```

Comment:
After merging in the `main` branch (0f6003f, which I believe has some changes to the table) and running

```bash
python utils/check_table.py --fix_and_overwrite
```

(`make fix-copies` doesn't work for me right now because `python utils/check_docstrings.py --fix_and_overwrite` raises an error, which seems to be due to a faulty jax/flax installation in my current environment), the script adds an entry for UnivNet but doesn't add a check mark for PyTorch support:

https://github.com/huggingface/transformers/blob/d32977c0554240896942e749968e18edcbea00d1/docs/source/en/index.md?plain=1#L264-L266

I have run into this issue before (see https://github.com/huggingface/transformers/pull/24799#issuecomment-1655150488) but still haven't figured out what causes this.

Reply:
The issue has been solved, seems like renaming `UnivNetGan` to `UnivNetModel` might have helped.